### PR TITLE
Add Nequix model

### PR DESCRIPTION
**Pull request by teddykoker**
## Description

The Nequix model is a compact and modified version of NequIP with added layer normalization, trained with the Muon optimizer, and implemented in JAX (full training pipeline and ASE calculator). The model is 700K parameters and can be trained in less than 500 GPU hours. I have uploaded the preprint [here](https://teddykoker.com/docs/nequix.pdf), which I which I will add to arxiv, once the Matbench results are verified. Furthermore, the full codebase for training our model can be found at https://github.com/atomicarchitects/nequix/. Please let me know if you need any additional information or modifications. The `test_<task>` scripts should run on a GPU machine with `uv run models/nequix/test_<task>.py`. Thanks in advance!

The submission also includes:

- a SLURM-ready discovery workflow producing per-job Nequix JSONL outputs
- a kappa workflow with per-structure logging and optional force-constant export
- an aggregation tool that joins predictions to WBM, applies energy corrections, and exports CSV/JSONL
- an interactive Nequix energy-parity plot

## Checklist

Please check the following items before submitting your PR:

- [X] I created a new folder and YAML metadata file `models/<arch_name>/<model_variant>.yml` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like author details, training set names and important hyperparameters.
- [X] I added the new model as a new attribute [`Model.<arch_name>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) on the `Model` enum in `enums.py`.
- [X] I uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<model_variant>-preds.csv.gz`).
- [X] I uploaded the model-relaxed structures file to Figshare or another cloud storage service in [JSON lines format](https://jsonlines.org) (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.jsonl.gz`). JSON Lines allows fast loading of small numbers of structures with `pandas.read_json(lines=True, nrows=100)` for inspection.
- [X] I uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [X] I included the urls to the Figshare files in the YAML metadata file (`models/<arch_name>/<model_variant>.yml`). If not using Figshare I have included the urls to the cloud storage service in the description of the PR.
- [X] I included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.

- [ ] I included a training script (`train_<arch_name>.py`) if I trained a model specifically for this benchmark.
- [ ] I included a `readme.md` with additional details about my model.

### CODE REVIEW

File: models/nequix/nequix-mp-1.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+doi: https://teddykoker.com/docs/nequix.pdf # TODO
+paper: https://teddykoker.com/docs/nequix.pdf
+pr_url: https://github.com/janosh/matbench-discovery/pull/276
+checkpoint_url: https://figshare.com/files/57245573
+
+openness: OSOD
+train_task: S2EFS
+test_task: IS2RE-SR
+targets: EFS_G
+model_type: UIP
```
[the remark this answers is not in the document]
Comment (teddykoker) [reply models/nequix/nequix-mp-1.yml:36]: Note, most other models use IS2RE-SR as the test task as well, e.g. https://github.com/janosh/matbench-discovery/blob/f52b4d99f06c70ec27a11638e3dd007ec8f3dee7/models/eSEN/eSEN-30m-mp.yml#L48

File: models/nequix/test_nequix_kappa.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            atoms.calc = None
+            atoms.constraints = None
+            atoms.info = init_info | atoms.info
+
+            # Check if symmetry was broken during relaxation
+            relaxed_spg = MoyoDataset(MoyoAdapter.from_atoms(atoms)).number
+            broken_symmetry = spg_num != relaxed_spg
+            relax_dict = {
+                "max_stress": max_stress,
+                "reached_max_steps": reached_max_steps,
```
[the remark this answers is not in the document]
Comment (teddykoker) [reply models/nequix/test_nequix_kappa.py:151]: The nequix calculator returns stress in voigt form, so the current code is correct.

**Review by janosh — APPROVED** [review]
the metrics i pushed in beda4d7 are consistent with what you report in https://teddykoker.com/docs/nequix.pdf
